adb/device-manager.js

```diff
@@ -50,9 +50,9 @@ export class DeviceManager {
     });
 
     ipcRenderer.on('portscan-requested', (event, args) => {
```

Comment:
You can define default values for Parameters `ipcRenderer.on('portscan-requested', (event, args = {} ) => {` then you dont need to assign the default value to args in the next line.

Reply:
Fancy, done 👍